src/features/Faucet/components/Error.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { FunctionComponent } from 'react';
+
+import styled from 'styled-components';
+
+import { InlineMessage } from '@components';
+import translate from '@translations';
+import { InlineMessageType } from '@types';
+import { formatSupportEmail } from '@utils';
+
+const Box = styled.div`
+  border: 1px solid #fa863f;
+  box-sizing: border-box;
+  box-shadow: inset 0px 1px 1px rgba(63, 63, 68, 0.05);
+  border-radius: 2px;
+  margin-top: 40px;
+`;
+
+const ContactUsText = styled(InlineMessage)`
+  font-style: italic;
+  font-size: 14px;
+  margin: 20px;
+  a {
+    color: var(--color-link-color);
```

Comment:
Use `COLORS` from `@theme` to define colors. SCSS shouldn’t be used since we want to get rid of it